lib/jstp.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
```

Comment:
IIUC, maybe rename the file to `provider-jstp-api.js` and `createApi` -> `createProviderApi`?

Reply:
@lundibundi, we use dots instead of dashes as word separators in filenames in this project.

Reply:
@lundibundi what do you think about `remote.provider.api.js` or `remote.provider.jstp.api.js` for the filename?

Reply:
+1 for `remote.provider.api.js`, yet I would prefer to leave `createAPI` as it as, because it would be obvious, that it creates API for the remote provider, based on a file it is located in.